packages/framework/src/Model/Customer/Mail/RegistrationMail.php

```diff
@@ -39,11 +40,11 @@ public function __construct(Setting $setting, DomainRouterFactory $domainRouterF
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Model\Customer\User $user
      * @param \Shopsys\FrameworkBundle\Model\Mail\MailTemplate $mailTemplate
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User $user
      * @return \Shopsys\FrameworkBundle\Model\Mail\MessageData
      */
-    public function getMessageDataByUser(User $user, MailTemplate $mailTemplate)
+    public function createMessage(MailTemplate $mailTemplate, $user)
```

Comment:
Are you sure that without class type `User` it will work?

Reply:
It is not better, but the interface has this function prototype, so the implementation must keep it